src/pages/Profile.jsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import StyledFieldset from '../components/StyledFieldset';
+
+export default function Profile() {
+  const profile = {
+    firstName: 'Joseph',
+    lastName: 'Tajaran',
+    email: 'joseph@tajaran.com',
+    pronouns: 'he/him',
+    skill: 'UX Design',
+  };
+
+  return (
+    <StyledFieldset>
```

Comment:
A fieldset should be used inside a form. 
This isn't really a form though. We're just displaying data.
Can this be restructured?